### PR TITLE
fixed few typos

### PR DESCRIPTION
**Pull request by bhavayAnand9**
Fixes few typos in README.MD file

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
Have you checked for any grammatical errors?

**Comment by bhavayAnand9** [comment]
I just took a quick look and found these. I'll check the doc again and create a PR if found any error.

**Comment by HarshKhandeparkar** [comment]
No need yo create a new PR.

File: README.md [hunk 1]
```diff
@@ -48,14 +48,14 @@ A diagram of this running 5 steps on a single sample image may help explain how
 This library works in the browser, in Node, and on the commandline (CLI), which we think is great.
```
Comment (HarshKhandeparkar) [inline README.md]: No space between command and line. (line 48)

File: README.md [hunk 2]
```diff
@@ -48,14 +48,14 @@ A diagram of this running 5 steps on a single sample image may help explain how
...
 
 ### Unix based platforms
-You can setup a local environement to test the UI with `sudo npm setup` followed by `npm start` 
+You can setup a local environment to test the UI with `sudo npm setup` followed by `npm start`
```
Comment (HarshKhandeparkar) [inline README.md]: Set up are two different words.

**Comment by HarshKhandeparkar** [comment]
Maybe you can use Grammarly to check for errors and fix them.

**Comment by bhavayAnand9** [comment]
Caught a few more.

File: README.md [hunk 3]
```diff
@@ -516,7 +516,7 @@ Image sequencer supports stringifying a sequence which is appended to the url an
 channel{channel:green},invert{}
 ```
 
-Sequencer also supports use of `()` in place of `{}` for backwards compatibility with older links.(This syntax is deprecated and should be avoided as far as possible)
+Sequencer also supports the use of `()` in place of `{}` for backward compatibility with older links. (This syntax is deprecated and should be avoided as far as possible)
```
Comment (HarshKhandeparkar) [inline README.md]: I guess backwards compatibility is more appropriate.

**Comment by HarshKhandeparkar** [comment]
Try not to apply all the changes that Grammarly suggests. There may be some technical words which are not quite correct grammatically.

**Comment by HarshKhandeparkar** [comment]
Nice work! You caught a lot of typos.

**Comment by bhavayAnand9** [comment]
> Try not to apply all the changes that Grammarly suggests. There may be some technical words/phrases which are not quite correct grammatically.

I don't think I did but If you find something let me know.

**Comment by bhavayAnand9** [comment]
No I didn't open an issue.

File: README.md [hunk 4]
```diff
@@ -236,10 +236,10 @@ modules.
 sequencer.run();
 ```
 
-Sequencer can be run with a custom config object
+The Sequencer can be run with a custom config object
```
Comment (jywarren) [inline README.md]: Here, we can skip "The"

File: README.md [hunk 5]
```diff
@@ -474,7 +474,7 @@ return value: **`sequencer`** (To allow method chaining)
 ### Inserting steps on an image
 
 The `insertSteps` method can also accept an `image_name` parameter. `image_name`
-may be an array. Everything else remains the same. JSON Inout is acceptable too.
+may be an array. Everything else remains the same. JSON Input is acceptable too.
```
Comment (jywarren) [inline README.md]: Here, 'input' should be lower case, thanks!

**Comment by bhavayAnand9** [comment]
Suggested changes applied.

**Comment by jywarren** [comment]
Awesome, thank you! We'd love help with something else too if you're interested!!!

**Comment by bhavayAnand9** [comment]
> can you tell us your Twitter handle so we can thank you properly?

My Twitter handle is 'AnandBhavay'

**Pull request merged**